internal/ui/gists.go

```
@@ -0,0 +1,457 @@
+		return s
+	}
+	if len(g.Files) > 0 && g.Files[0].Name != "" {
+		return g.Files[0].Name
+	}
+	return g.Name
+}
+
+// selectedGist returns the gist at the cursor inside the
+// sorted-filtered view, so the action menu never reimplements the row
+// pipeline.
+func (gm GistsModel) selectedGist(stats *github.Stats) (github.Gist, bool) {
```

golangci-lint (2.12.2)

[error] func GistsModel.selectedGist is unused

(unused)